modules/comment.js

```diff
@@ -0,0 +1,41 @@
+import { getComments, postComment } from '../src/apiCall.js';
+
+const showPopup = (result) => {
+  const movieTitle = document.querySelector('.movie-title');
+  const movieImage = document.querySelector('.movie-image');
+  const movieSummary = document.querySelector('.movie-summary');
+  const modalContainer = document.querySelector('.modalContainer');
+  document.querySelector('.form').id = result.id;
+  let summary = '';
+  if (result.summary === null) {
+    summary = 'The description for this movie is not available yet ...';
+  } else {
+    summary = result.summary.replace('<p>', '').replace('</p>', '').replace('<b>', '').replace('</b>', '');
+  }
+  movieTitle.textContent = result.name;
+  movieImage.src = result.image.medium;
+  movieSummary.textContent = summary;
+  modalContainer.style.display = 'flex';
+};
+
+const addNewComment = () => {
+  const form = document.querySelector('.form');
+  const commentsCount = document.querySelector('.comments');
+  const commentsList = document.querySelector('.comments-list');
+  const inputName = document.querySelector('.input-name');
+  const inputText = document.querySelector('.input-text');
+
+  form.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    await postComment(form.id, inputName.value, inputText.value);
+    form.reset();
+    const allComments = await getComments(form.id);
+    allComments.forEach((item) => {
+      commentsList.innerHTML
+        += `<li>${item.creation_date} ${item.username}: ${item.comment}</li>`;
+    });
+    commentsCount.textContent = `Comments (${commentsList.children.length})`;
+  });
```

Comment:
- [ ] When I tried to comment on one movie. Initially it was 0, but when I commented I got 2 comment. Again when commented, i got 5. I think the issue is they are fetching and adding all the comments again to the previous list. You can either clear the previous comment while fetching so that you get the correct list in the UI as well. :slightly_smiling_face:

### Intitally, when no comment is there.
![Screenshot from 2022-04-23 19-44-27](https://user-images.githubusercontent.com/22434660/164909870-6c1d7fda-88af-416f-9cab-83d2d0dad11d.png)

### When I put one comment

![Screenshot from 2022-04-23 19-44-41](https://user-images.githubusercontent.com/22434660/164909896-a3cd7b03-6a83-4d33-8f26-2cc7f1c532df.png)

### When I put another

![Screenshot from 2022-04-23 19-45-22](https://user-images.githubusercontent.com/22434660/164909910-17159de4-cf11-4b78-831c-c9e9237f0178.png)